### DISCUSSION TITLE
disable object repetition

### DISCUSSION THREAD

**Discussion by byfaruk** [codeproject/CodeProject.AI-Server#232] [Q&A]
Whenever there is movement, a photo comes from my camera. And he perceives it as a new object each time. The AI ​​will not notify me if the object's location does not change or a new object appears. How do I configure this (disable object repetition)

**Comment by tbc725** [comment]
This part is up to your surveillance system's setting. For example, in the AgentDVR, there is a "prevent repeat" setting in their Object Detection menu. I don't know which software you are using but it might have such setting in it.
